source/quick-start.txt

```diff
@@ -14,6 +14,52 @@ Quick Start
 
 .. include:: /includes/quick-start/overview.rst
 
+Create a MongoDB Cluster
+------------------------
+
+Set Up a Free Tier Cluster in Atlas
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Complete the
+:atlas:`Get Started with Atlas </getting-started>` guide to set up a new
+Atlas account and free tier MongoDB cluster, load datasets, and
+interact with the data.
+
+After completing the steps in the Atlas guide, you have a new MongoDB
+cluster deployed in Atlas, a new database user, and
+:atlas:`sample datasets loaded </sample-data/>` into your cluster.
+
+Find Your Connection String
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To connect to the Atlas cluster you created in the previous step, you need the 
+*connection string* for your cluster and user. A connection string contains the 
+information the {+driver-short+} needs to connect 
+to a MongoDB cluster, including the hostname or IP address and port of the
+cluster, authentication mechanism, user credentials (when applicable), and
+other connection options.
+
+To retrieve the connection string for your Atlas cluster and user, complete the 
+:guides:`Get Connection String </atlas/connection-string>`
+guide. Once you have finished the guide, you will have a connection string similar to the 
+following:
+
+.. code-block:: bash
+
+   "mongodb+srv://<username>:<password>@cluster0.abc.mongodb.net/?retryWrites=true&w=majority"
+
+Run the following code at the command prompt to save your Atlas connection string to an 
```

Comment:
Suggestion:

What do you think about creating a new section for this part, perhaps entitled "Set Your Connection String Environment Variable"? That helps logically separate it from "Find Your Connection String".

If a user reads https://www.mongodb.com/docs/guides/atlas/account/ (suggested link in an earlier comment) to completion, they should be able to start with their "connection string in [their] copy buffer".